IPython/core/interactiveshell.py

```diff
@@ -391,6 +391,13 @@ def _prompt_trait_changed(self, name, old, new):
 
     history_length = Integer(10000, config=True)
 
+    history_load_length = Integer(1000, config=True, help=
+        """
+        Determines the number of saved history entries to be loaded
```

Comment:
Documentation style point - I'd prefer to leave out the word 'Determines', so this would start 'The number of ...'.